pages.th/windows/cls.md

```diff
@@ -1,7 +1,7 @@
 # cls
 
 > ล้างตัวอักษรทั้งหมดที่อยู่บนหน้าต่าง
-> ข้อมูเพิ่มเติม: https://learn.microsoft.com/windows-server/administration/windows-commands/cls
+> ข้อมูลเพิ่มเติม: https://learn.microsoft.com/windows-server/administration/windows-commands/cls
```

Comment:
We use this in [set-more-info-link.py](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py). Check out the 39th line in set-more-info-link.py.
```suggestion
> ดูเพิ่มเติม: https://learn.microsoft.com/windows-server/administration/windows-commands/cls
```

Reply:
> We use this in [set-more-info-link.py](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py). Check out the 39th line in set-more-info-link.py.

@blueskyson it seems the current term means **See also: (ดูเพิ่มเติม:)**, and the correct term over there must be **More Information( ข้อมูลเพิ่มเติม:)** it seems. I created a new PR to update all pages #8585. I will include this too.